Feat/salva id sentimentos

ThaisMap opened the pull request:
- novos registros de sentimentos no diário serão salvos com id
- registros antigos que forem atualizados também vão ser salvos com id
- adiciona uma flag no diário para sentimentos salvos com id, para ajudar na migração

Finish #178935830

ThaisMap committed:
feat: altera retorno do registro

- Os sentimentos retornados no registro do diário vão conter os sentimentos baseado na collection de sentimentos do usuário, ou nos sentimentos modelo, caso o usuário não tenha a collection ainda

ThaisMap committed:
feat: usa collection sentimento nos componentes

- Altera componente de sentimento com emoji e checkbox group e gráficos de sentimentos para usar os dados da collection e não os hard coded
- Ajusta as paginas onde esses componentes são usados, para fornecer os parâmetros corretos

ThaisMap committed:
feat: salva id dos sentimentos no diário

- novos registros de sentimentos no diário serão salvos com id
- registros antigos que forem atualizados também vão ser salvos com id
- adiciona uma flag no diário para sentimentos salvos com id, para ajudar na migração

kellynvd commented on src/components/diario/SentimentosCheckboxGroup.tsx:
```diff
@@ -45,15 +47,28 @@ const useStyles = makeStyles(() =>
   const classes = useStyles()
+  const [sentimentos, setSentimentos] = useState([])
+  useEffect(() => {
+    const getSentimentosUsuario = async () => {
```
Acho que isso isso pode estar em uma função, tipo a dos hábitos (`getGrupoDeHabitosIniciais`) pra não deixar essa lógica no componente. O que acha?

kellynvd commented on src/services/sentimentos/GetSentimentosByUserId.ts:
```diff
@@ -0,0 +1,30 @@
+      ? SentimentosDoUsuario
+      : await new GetAllSentimentosModelos().call()
+
+    return Sentimentos.map(sentimento => new Sentimento(sentimento))
```
Acho que aqui o ideal seria o repository já retornar os **Sentimentos** instanciado e não o service fazer isso

kellynvd reviewed (changes requested):
Deixei algums comentários e tem mais 2 pontos:

- [x] 1- Deveria estar funcionando normalmente mesmo sem a migração? Porque acessei com meu user e está dando erro:

![Screenshot from 2021-07-28 19 30 51](https://user-images.githubusercontent.com/13313889/127404749-54bb2599-18fd-4504-9a44-68a04583ddd2.png)

Na develop não da erro, então acredito que esteja relacionado com essas alterações mesmo.
user: _kellynvd@gmail.com_
senha: _123321_

- [x] 2- Ficou alguns console.log() no código, Não comentei em todos então tem que dar uma revisada :)

ThaisMap committed:
fix: erro em map e outros ajustes

- correção do map em grupo de hábitos sem hábito (como no hotfix #108)
- função para pegar os sentimentos modelos ou do usuário movida para utils
- repositories já retornam Sentimentos, ao invés de converter no service
- removidos alguns console.log

ThaisMap committed:
fix: erro de tipagem nos graficos
